pkg/subctl/cmd/deploybroker.go

```diff
@@ -49,7 +49,15 @@ func init() {
 	deployBroker.PersistentFlags().StringVar(&ipsecSubmFile, "ipsec-psk-from", "",
 		"Import IPsec PSK from existing submariner broker file, like broker-info.subm")
 
-	deployBroker.PersistentFlags().BoolVar(&serviceDiscovery, "service-discovery", false,
+	deployBroker.PersistentFlags().Bool("service-discovery", true,
+		"Enable Multi-Cluster Service Discovery")
```

Comment:
So we’re dropping support for `--service-discovery=false`?

Reply:
I'm changing it back to that... another patch in a minute.